knative-operator/cmd/manager/main.go

```diff
@@ -111,6 +112,9 @@ func main() {
 	hookServer.KeyName = "apiserver.key"
 	hookServer.CertName = "apiserver.crt"
 
+	disableHTTP2 := func(c *tls.Config) { c.NextProtos = []string{"http/1.1"} }
```

Comment:
Probably we could have a flag here to enable/disable on demand like in other operators.